### DISCUSSION TITLE
Struggle to make it work - ollama url not saved

### DISCUSSION THREAD

**Discussion by HyperUpscale** [jonfairbanks/local-rag#56] [Q&A]
Running the docker image... 

I put:
 extra_hosts:
- 'host.docker.internal:host-gateway'

But still in the Ollama Endpoint the url is : "http://localhost:11434"
When I manually edit it to: "http://host.docker.internal:11434" then it finds the models and seem to start working , but doesn't keep that setting.

🗂️  GitHub Repo

Select a GitHub.com repo

Processing...

✔️ LLM Initialized

✔️ Embedding Model Created

FileNotFoundError: [Errno 2] No such file or directory: '/home/appuser/data'
Traceback:
File "/home/appuser/utils/rag_pipeline.py", line 119, in rag_pipeline
    documents = llama_index.load_documents(save_dir)
File "/home/appuser/utils/llama_index.py", line 99, in load_documents
    for file in os.scandir(data_dir):
    
    
![image](https://github.com/jonfairbanks/local-rag/assets/126105457/ca1c6332-abf4-42c7-9477-cc947a96c8d2)

I cant make it work.

**Comment by jonfairbanks** [comment]
Please read the [troubleshooting guide](https://github.com/jonfairbanks/local-rag/blob/develop/docs/troubleshooting.md). Also based on the directory in the stack trace, you are using Windows. Please use a Linux host if possible.
